python/lsst/analysis/tools/actions/plot/histPlot.py

```diff
@@ -536,7 +541,7 @@ def _calcStats(self, data):
         mad = sigmaMad(data)
         return num, med, mad
 
-    def _addReferenceLines(self, ax, panel, panel_range, legend_font_size=7):
+    def _addReferenceLines(self, ax, panel, panel_range, meds, legend_font_size=7):
```

Comment:
Should meds be a kwarg? Looks like it is optional later.

Reply:
I think it makes more sense to keep it a required input. Even though it is only used if the `refRelativeToMedian` flag is set, I'd rather be sure it's there when that flag is set than have an error when the flag is set and `meds` is _not_ passed.